src/preprocessing/preprocess.ts

```diff
@@ -0,0 +1,24 @@
+import { Config } from '../config';
+import { modules as preprocessingModules } from './preprocessing-modules/index';
+
+/**
+ * Runs the preprocessing modules on the given AMP code.
+ *
+ * @param {string} amp AMP code to preprocess
+ * @param {!Config} config Global config
+ * @param {Array<string>=} modules Module whitelist (leave empty to run all)
+ * @return {!Promise<string>} Preprocessed AMP code
+ */
+export async function preprocessAMP(
+  amp: string,
+  config: Config
+): Promise<string> {
+  const moduleSet = new Set(config.skipPreprocessingModules || []);
+  for (const module of preprocessingModules) {
+    if (preprocessingModules && !moduleSet.has(module.name)) {
```

Comment:
This should change to 
    If ( preprocessingModules && moduleSet.has(module.name))

We want to skip if the module to skip is encountered.

Reply:
Ouch, good catch. Fixed.